README.md
<p align="center">
  <a href="https://ant-design-colorful.github.io/ant-design-colorful/">
    <img width="200" src="https://avatars1.githubusercontent.com/u/73024040?s=200&v=4">
  </a>
</p>

<h1 align="center">ant-design-colorful</h1>

<div align="center">

以 ant-design 为基础衍生的多彩组件

[![](https://img.shields.io/badge/based%20on-ant--design-%231890ff?style=flat-square)](https://github.com/ant-design/ant-design)
[![](https://img.shields.io/badge/docs%20by-dumi-%239254de?style=flat-square)](https://github.com/umijs/dumi)
[![](https://img.shields.io/github/contributors/ant-design-colorful/ant-design-colorful?style=flat-square&color=%23389e0d)](https://github.com/ant-design-colorful/ant-design-colorful/graphs/contributors)
![](https://img.shields.io/github/last-commit/ant-design-colorful/ant-design-colorful?color=%2308979c&style=flat-square)
![](https://img.shields.io/github/stars/ant-design-colorful/ant-design-colorful?style=flat-square&color=%23ff7875)
[![](https://img.shields.io/npm/v/antd-colorful?color=1d39c4&style=flat-square)](https://www.npmjs.com/package/antd-colorful)
[![](https://img.shields.io/npm/dt/antd-colorful?color=ff69b4&style=flat-square)](https://www.npmjs.com/package/antd-colorful)

</div>

## 📒 预览

https://ant-design-colorful.github.io/ant-design-colorful/

## ✨ 特性

- 📦 简单易用
- 🌈 保持 ant-design 写法，对应 `4.8.0`
- 🎨 绚丽多彩

## 📍 如何使用

### 安装

```bash
npm i antd-colorful
## or
yarn add antd-colorful
```

### 使用

```js
import 'antd-colorful/index.css';
```

### 说明

- 如果想在自己项目中引用，可查看 Demo 写法，建议单独查看代码引入单独样式。
- 由于 特定属性 / 类型引起 `css` 变动而出现样式未生效或异常，建议采用 Demo 写法，或自行参照修正。
- 正逐步构想并添加，如果你有好的想法和建议，欢迎 PR ~

## 📐 组件看板

- Breadcrumb 面包屑
  - 多彩颜色
  - 组件结合
- Button 按钮
  - 多彩背景
  - 涟漪效果
  - 交互效果
  - 键盘按键
  - 图标动画
- Divider 分割线
  - 多彩颜色
  - 类型扩展
  - 多重展示
- Pagination 分页
  - 多彩颜色
  - 简洁实心
  - 基础拼接
- Radio 单选框
  - 多彩颜色
  - 交互效果

## 🖥 兼容环境

- 现代浏览器和 IE11（需要 [polyfills](https://ant.design/docs/react/getting-started-cn#兼容性)）。
- [Electron](https://www.electronjs.org/)

| [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png" alt="IE / Edge" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)<br>IE / Edge | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png" alt="Firefox" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)<br>Firefox | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" alt="Chrome" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)<br>Chrome | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png" alt="Safari" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)<br>Safari | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/electron/electron_48x48.png" alt="Electron" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)<br>Electron |
| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| IE11, Edge                                                                                                                                                                                                     | last 2 versions                                                                                                                                                                                                  | last 2 versions                                                                                                                                                                                              | last 2 versions                                                                                                                                                                                              | last 2 versions                                                                                                                                                                                                      |

## LICENSE

[MIT](https://github.com/ant-design-colorful/ant-design-colorful/blob/main/LICENSE)


CHANGELOG.md
# antd-colorful

- 对应 `antd` 版本可在 `README.md` 中查看
- 支持组件可在 `README.md` 中查看

## 0.0.2

`2020-11-10`

- 📖 更新 README.md

## 0.0.1

`2020-11-10`

- 🎊 首次打包。


.umirc.ts
export default {
  title: 'ant-design-colorful',
  mode: 'site',
  favicon: 'https://avatars1.githubusercontent.com/u/73024040?s=200&v=4',
  logo: 'https://avatars1.githubusercontent.com/u/73024040?s=200&v=4',
  outputPath: 'docs-dist',
  hash: true,
  base: '/ant-design-colorful/',
  publicPath: '/ant-design-colorful/',
  resolve: {
    includes: ['docs'],
  },
  theme: {
    '@c-primary': '#1890ff',
  },
  menus: {
    '/guide': [
      {
        children: ['guide/index', 'guide/how-to-use'],
      },
    ],
  },
  navs: [
    { title: '介 绍', path: '/guide' },
    { title: '组 件', path: '/components' },
    {
      title: 'GitHub',
      path: 'https://github.com/ant-design-colorful/ant-design-colorful',
    },
  ],
  extraBabelPlugins: [
    [
      'import',
      {
        libraryName: 'antd',
        libraryDirectory: 'es',
        style: 'css',
      },
    ],
  ],
};
